recipe/meta.yaml

```diff
@@ -15,7 +15,7 @@ source:
   {{ hash_type }}: {{ hash_val }}
 
 build:
-  noarch: python
+  skip: true  # [py<37]
```

Comment:
The package can work with any `python >=2.7` so there is no sense to skip explicitly